### DISCUSSION TITLE
v1.15.8

### DISCUSSION THREAD

**Discussion by lwindolf** [lwindolf/liferea#1383] [General]
## What's Changed

* #1289 Solve some Webkit deprecations
	(Lars Windolf)

* Fixes #1359: Prevent passing "" as CLI argument causing a useless subscription.
	Also checks for duplicates when subscribing from CLI/DBUS.
	(Lars Windolf)

* Fixes #1372: Fix feed icon not parsing if the URL doesn't have a file extension bug 
	(Stomern)

* Fixes #1382: Doesn't recognize atom:link element with no rel attribute bug 
	(Lars Windolf, reported by Ed Davies)

* Update of Turkish translation (emintufan)

## New Contributors

* @Stomern made their first contribution in https://github.com/lwindolf/liferea/pull/1372

**Full Changelog**: https://github.com/lwindolf/liferea/compare/v1.15.7...v1.15.8

<hr /><em>This discussion was created from the release <a href='https://github.com/lwindolf/liferea/releases/tag/v1.15.8'>v1.15.8</a>.</em>

**Comment by genodeftest** [comment]
I'm slightly confused: Is this a stable release or is it not?

[Fedora has updated to 1.15.8 already](https://packages.fedoraproject.org/pkgs/liferea/liferea/), but the [flatpak is still on 1.14.6](https://flathub.org/apps/net.sourceforge.liferea). Which of those is correct?

**Reply by lwindolf** [reply]
1.15.x is considered unstable. In the Releases section the naming is consistent.

Some distros ignore the odd numbered releases to be unstable though...

So from my pov flatpak is correct.

**Reply by genodeftest** [reply]
Thanks for the clarification :-)
